server/src/test/java/io/unitycatalog/server/sdk/tables/SdkTableOperations.java

```diff
@@ -18,6 +21,18 @@ public SdkTableOperations(ApiClient apiClient) {
   }
 
   public TableInfo createTable(CreateTable createTableRequest) throws ApiException {
+    if (createTableRequest.getTableType() == TableType.MANAGED) {
+      // Caller actually should not set storage location for MANAGED tables. This function will set
+      // it to the staging location of staging table.
+      assert createTableRequest.getStorageLocation() == null;
+      CreateStagingTable createStagingTableRequest =
+          new CreateStagingTable()
+              .catalogName(createTableRequest.getCatalogName())
+              .schemaName(createTableRequest.getSchemaName())
+              .name(createTableRequest.getName());
+      StagingTableInfo stagingTableInfo = tablesApi.createStagingTable(createStagingTableRequest);
```

Comment:
So for every normal table creation process, it requires us to create the staging table firstly ?  Could you please explain more about ? Or is there any reference that I can read ?